### ISSUE TITLE
[Feature Request] How to customize the game cards displayed on the home page in game mode

### ISSUE THREAD

**Issue by RosenYin** [ValveSoftware/steam-for-linux#10463]
#### Your system information

* Steam client version:1705108172
* SteamOS version: 3.5.7
* Opted into Steam client beta?: [Yes/No] No
* Opted into SteamOS beta?: [Yes/No] No
* Have you checked for updates in Settings > System?: [Yes/No] Yes

#### Question about the game cards displayed on the homepage sometimes not being the games I want displayed:

Sometimes the homepage will display games that I have played on the PC or newly added, but sometimes these games are completely incompatible with steamdeck, such as The finals.

But they will still be displayed on my homepage. This display method is I think it's a little inefficient to select the game I want to play because it's annoying to see the game you want to play on the homepage only to find out it's not compatible and then there's no way you can stop it from showing up.

I would like to be able to add the ability to customize which games are displayed on the homepage, so I can put the ones I want to display in a newly created collection and it will always display what I want it to display.

**Comment by Gallocon** [comment]
Yes. As a neurodivergent person, decision paralysis is something I deal with. Even with the 20 games displayed on the Home Screen, it can be overwhelming. And some times I just don't play anything. 

If I could set a custom list of games (instead of it being solely _recent games_) the Deck experience would be so much better.
